appinventor/appengine/src/com/google/appinventor/client/Ode.java

```diff
@@ -186,6 +186,14 @@ public class Ode implements EntryPoint {
 
   private Random random = new Random(); // For generating random nonce
 
+  // Wether or not to show the "Delete Account" Menu public versions
+  // may want this while those managed by a "portal" or other
+  // management system may not want it It is set by the
+  // auth.deleteaccountallowed flag. The default is true.
+
+  private boolean deleteAccountAllowed;
```

Comment:
In other places we reference the flag coming from the config rather than storing it on the Ode object (see `getGalleryReadOnly`, for example). Is there much value in also storing the flag here rather than simply calling `config.getDeleteAccountAllowed()`?

Reply:
I took this suggestion